linode/manager
# refactor: [M3-6357] - Refactor LinodeCLIModal

## Description 📝
MUI v5 migration for the `LinodeCLIModal` component.

## Major Changes 🔄
- Ran `tss-react` codemod on `LinodeCLIModal`
- Replaced `default export` with `named export`.

## How to test 🧪
1. Verify e2e tests pass
2. Verify unit tests pass
3. Inspect for any visual regressions
